### DISCUSSION TITLE
Type error : File is not a module. Upon changing the contents in .next folder, it goes back to the same when I do npm run build

### DISCUSSION THREAD

**Discussion by Monica0925** [vercel/next.js#62094] [Help]
### Summary

![image](https://github.com/vercel/next.js/assets/76522123/21336933-29c1-45f1-a18a-dc72d208b8dd)

I'm working on a nextjs , react js and typescript project. (pretty new to web dev)
Tried doing npm run dev which works just fine but when I do npm run build it gives the above error.
Tried changing the import statement but it goes back to the original statement when I do npm run build.

**Comment by mattjohnpowell** [comment]


**Reply by ciruz** [reply]
@mattjohnpowell whats the problem, do you have some code to share?

**Reply by mattjohnpowell** [reply]
Really dumb mistake on my part. (There was an empty file with a very similar name to a non-empty file. deleted the empty one and all good)
